notebooks/106-auto-device/106-auto-device.ipynb

```diff
@@ -0,0 +1,407 @@
+    "\n",
+    "Please follow the [104-model-tools](../104-model-tools/README.md) to download googlenet-v1 and convert (--precisions FP16), copy the results (googlenet-v1.bin and googlenet-v1.xml) to notebooks\\106-auto-device\\model\\ folder"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "740bfdd8",
+   "metadata": {},
+   "source": [
+    "## compile_model with AUTO device\n",
+    "### Default behavior of compile_model without device_name\n",
+    "By default compile_model will select AUTO as device_name if it is not specificed."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "73d7aedb",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import cv2\n",
+    "import matplotlib.pyplot as plt\n",
+    "import numpy as np\n",
+    "from openvino.runtime import Core, CompiledModel\n",
+    "\n",
+    "ie = Core()\n",
+    "\n",
+    "model = ie.read_model(model=\"model/googlenet-v1.xml\")\n",
+    "compiled_model = ie.compile_model(model=model, config={\"LOG_LEVEL\":\"LOG_INFO\"})\n",
+    "\n",
+    "if isinstance(compiled_model, CompiledModel):\n",
+    "    print(\"Compile model without device_name successfully.\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "85b48949",
+   "metadata": {},
+   "source": [
+    "### compile_model with AUTO as device name"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "0814073d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "compiled_model = ie.compile_model(model=model, device_name=\"AUTO\")\n",
+    "\n",
+    "if isinstance(compiled_model, CompiledModel):\n",
+    "    print(\"Compile model with AUTO successfully.\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "705ce668",
+   "metadata": {},
+   "source": [
+    "## First inference latency benifit with AUTO\n",
+    "One of the key performance benefits of AUTO is on first inference latency (FIL = compile model time + fist inference execution time). Directly using CPU device would produce the shortest first inference latency as the OpenVINO graph representations can really quickly be JIT-compiled to CPU. The challenge is with the GPU. Since the OpenCL complication of graph to GPU-optimized kernels takes a few seconds to complete for this platform. If AUTO selects GPU as the device, this initialization time may be intolerable to some applications, which is the reason for AUTO to transparently use the CPU as the first inference device until GPU is ready. \n",
```

Comment:
```suggestion
    "One of the key performance benefits of AUTO is on first inference latency (FIL = compile model time + fist inference execution time). Directly using CPU device would produce the shortest first inference latency as the OpenVINO graph representations can really quickly be JIT-compiled to CPU. The challenge is with the GPU. Since the OpenCL complication of graph to GPU-optimized kernels takes a few seconds to complete for this platform. If AUTO selects GPU as the device, this initialization time may be intolerable to some applications, which is the reason for AUTO to transparently use the CPU as the first inference device until GPU is ready. \n",
```
Please include URLs where the notebook end-user could find more information about this.